cardano-testnet/src/Testnet/Shelley.hs

```diff
@@ -116,6 +131,41 @@ rewriteGenesisSpec testnetOptions startTime =
 startTimeOffsetSeconds :: DTC.NominalDiffTime
 startTimeOffsetSeconds = if OS.isWin32 then 90 else 15
 
+
+mkTopologyConfig :: Int -> [Int] -> Int -> Bool -> ByteString
```

Comment:
This function has been duplicated, why?

Reply:
I am not sure right now (the PR was a team effort), I'll look into it.

Reply:
It's not a duplicated function, it has a different type signature and the meaning of its arguments is slightly different.  It's probably cleaner to leave this as is and not generalise further.